mathesar/models.py

```diff
@@ -222,6 +222,7 @@ class DataFile(BaseModel):
     created_from = models.CharField(max_length=128, choices=created_from_choices.choices)
     table_imported_to = models.ForeignKey(Table, related_name="data_files", blank=True,
                                           null=True, on_delete=models.SET_NULL)
+    base_name = models.CharField(max_length=128)
```

Comment:
We need to ensure that any generated names are smaller than 63 bytes since that's how long Postgres identifiers can be.

https://www.postgresql.org/docs/current/limits.html